product_pricelist_fixed_price/README.rst

```diff
@@ -0,0 +1,27 @@
+Fixed price on pricelist rule
```

Comment:
Update this README with latest updates (AGPL-3 flag and bug tracker section): https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst